CMakeLists.txt

```diff
@@ -393,6 +397,17 @@ elseif(FLB_RELEASE)
   set(CMAKE_BUILD_TYPE "RelWithDebInfo")
 endif()
 
+if(FLB_RELEASE)
```

Comment:
I would suggest adding another option to let the user skip this feature (even if is inside a supported environment).

Reply:
Hi @edsiper, 
I added a new option with the default value `ReleaseOnly`.